src/draw/vg_lite/lv_draw_vg_lite_label.c

```diff
@@ -137,7 +137,7 @@ static void draw_letter_bitmap(lv_draw_vg_lite_unit_t * u, const lv_draw_glyph_d
     lv_vg_lite_buffer_init(
         &src_buf,
         dsc->bitmap,
-        lv_area_get_width(&image_area),
+        lv_vg_lite_width_align(lv_area_get_width(&image_area)),
```

Comment:
VG-Lite has this super non convenient width align requirement. We should put the logic to `lv_vg_lite_buffer_init ` and make sure stride is enough for aligned `width`.